src/lib/mojaloop-requests/toString.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I would much rather this was a func in the main mojaloopRequests.js file. I think making it a require is not necessary and makes the code harder to follow i.e. someone has to open the dependency. It is not used anywhere else, so again, I think it would be better in the main file.